.travis.yml

```diff
@@ -51,17 +51,18 @@ before_install:
 
     # when it's a release branch or tag
     if [[ "${TRAVIS_BRANCH}" =~ ${MULTIPASS_RELEASE_BRANCH_PATTERN} \
+          || "${TRAVIS_PULL_REQUEST_BRANCH}" =~ ${MULTIPASS_RELEASE_BRANCH_PATTERN} \
```

Comment:
This is getting hard to follow. Right now the branching is all intertwined:

```
if branch...
  if even_type...
    if branch
elif event... and branch...
  if branch...
elif event...
```

And `TRAVIS_PULL_REQUEST_BRANCH` itself depends on the event type. 

Could this be reorganized/simplified, so that tests for even type would be in one level, tests for branch in another, etc.? To get closer to a table structure, with non-overlapping cases and sub-cases.

Reply:
Yeah you're right, I'll rework.

Reply:
Cool, thanks!

Reply:
As discussed on IRC, we'll leave it as a separate step, ideally including a test suite that would run as a first step.